tests/infra/server.py

```diff
@@ -79,3 +93,98 @@ def shutdown(self):
 
 # noinspection PyProtectedMember
 ThreadedLocalServer._inject_api_requests_methods()
+
+
+class MockFusilladeHandler(BaseHTTPRequestHandler):
+    """
+    Create a mock Fusillade auth server endpoint so that any operation that tries to check
+    permissions with Fusillade will be handled correctly; we keep it simple and accept/reject
+    based on whether the principal (user) is on the whitelist or not.
+    """
+    _server = None
+    _thread = None
+    _whitelist = [
```

Comment:
If these service accounts are needed to validate tests, then this list should probably be populate from the ssm environment whitelist (this will also take of adding the valid service accounts for `staging` and `prod` which are missing here, as well as anyone on the team running locally).

Reply:
Good idea - using `get_admin_emails()` from `dss/operations/lambda_params.py`